reference/docs-conceptual/samples/Working-with-Printers.md

```diff
@@ -1,27 +1,31 @@
 ---
-ms.date:  06/05/2017
+ms.date:  12/23/2019
 keywords:  powershell,cmdlet
 title:  Working with Printers
 ---
 # Working with Printers
 
-You can use Windows PowerShell to manage printers by using WMI and the WScript.Network COM object from WSH. We will use a mix of both tools to demonstrate specific tasks.
+You can use Windows PowerShell to manage printers by using WMI and the **WScript.Network** COM
```

Comment:
Remove "Windows" reference. There should be a mention that WSH, as a COM object, is Windows only, though.

Reply:
Fixed